android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="co.psychedelic.plug">
-
+  package="co.psychedelic.plug"
+  android:installLocation="preferExternal">
```

Comment:
is this needed?

Reply:
i think you were trying to make the app work in M1 emulator when you added this

Reply:
It should fix the not enough storage problem in simulator

Reply:
maybe, but thats not something we want in the production app. In my case, using a Nexus emulator solved the "Insufficient space" issue